api/v1alpha1/common_types.go

```diff
@@ -72,6 +75,11 @@ const (
 	// ReasonRetrievalFailed indicates that dependent resource(s) could not be retrieved for further processing
 	ReasonRetrievalFailed ConditionReason = "RetrievalFailed"
 
+	// ReasonNoPreBackupPodsFound is given when no PreBackupPods are found in the same namespace
+	ReasonNoPreBackupPodsFound ConditionReason = "NoPreBackupPodsFound"
+	// ReasonWaiting is given when PreBackupPods are waiting to be started
+	ReasonWaiting ConditionReason = "Waiting"
```

Comment:
My thought was that this reason would be reusable for other conditions also (similar to "ready", "succeeded" or "failed"). Since currently this reason is only used in the `PreBackupPodReady` condition, this could be renamed. But OTOH, it should be clear enough from the condition's name and the message that the backup is waiting for a prebackup.

`NoPreBackupPodsFound` is the only reason the term `PreBackup` is mentioned, since none of the other reasons fit. 
But I have no hard feelings here.